go.mod

```diff
@@ -49,3 +53,5 @@ replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.0+incompatible
 	k8s.io/client => github.com/kubernetes-client/go v0.0.0-20190928040339-c757968c4c36
 )
+
+replace github.com/dapr/components-contrib => ../components-contrib
```

Comment:
Needed for it to build before the components-contrib side is in, won't be there once that side is merged.

Reply:
It's here https://github.com/dapr/components-contrib/pull/434